python/tests/unit/arcticdb/version_store/test_normalization.py

```diff
@@ -620,3 +620,17 @@ def test_norm_failure_error_message(lmdb_version_store_v1):
                [write_exception, batch_write_exception])
     assert all("pickle_on_failure" not in str(e.value) for e in
                [append_exception, batch_append_exception, update_exception])
+
+def test_bools_are_pickled(lmdb_version_store_allows_pickling):
```

Comment:
and numpy arrays?

Is there a test that Python bools and numpy arrays blow up like we expect when pickling is disabled?

Reply:
Arrays are pickled. I added tests, just in case.